enclave_wrangler/objects_api.py

```diff
@@ -324,44 +272,183 @@ def update_db_with_new_objects(objects=None):
+    return obj
+
+
+def fetch_cset_version(object_id: int) -> Dict:
+    """Get object from enclave"""
+    return fetch_object_by_id('OMOPConceptSet', object_id, 'codesetId')
+
+
+def fetch_cset_container(object_id: int) -> Dict:
+    """Get object from enclave"""
+    return fetch_object_by_id('OMOPConceptSetContainer', object_id, 'conceptSetId')
+
+
+def fetch_cset_member_item(object_id: int) -> Dict:
+    """Get object from enclave"""
+    return fetch_object_by_id('OMOPConcept', object_id, 'conceptId')
+
+
+def fetch_concept(object_id: int) -> Dict:
+    """Get object from enclave"""
+    return fetch_cset_member_item(object_id)
+
+
+def fetch_cset_expression_item(object_id: int) -> Dict:
+    """Get object from enclave"""
+    return fetch_object_by_id('OmopConceptSetVersionItem', object_id, 'itemId')
+
+
+def fetch_object_and_add_to_db(con: Connection, table: str, object_type_name: str, object_id: Union[int, str]) -> Dict:
+    """Fetch object and add to db"""
+    obj: Dict = fetch_object_by_id(object_type_name, object_id)
+    table_obj: Dict = convert_row(object_type_name, table, obj)
+    insert_from_dict(con, table, table_obj)
+    return table_obj
+
+
+def concept_set_container_enclave_to_db(con: Connection, object_id: str) -> Dict:
+    """Given ID, get object's current state from the enclave, and add it the DB"""
+    return fetch_object_and_add_to_db(con, 'concept_set_container', 'OMOPConceptSetContainer', object_id)
+
+
+def code_sets_enclave_to_db(con: Connection, object_id: int) -> Dict:
+    """Given ID, get object's current state from the enclave, and add it the DB"""
+    return fetch_object_and_add_to_db(con, 'code_sets', 'OMOPConceptSet', object_id)
+
+
+def cset_version_enclave_to_db(con: Connection, object_id: int) -> Dict:
+    """Alias for: code_sets_enclave_to_db()"""
+    return code_sets_enclave_to_db(con, object_id)
+
+
+def concept_set_version_item_enclave_to_db(con: Connection, object_id: str) -> Dict:
+    """Given ID, get object's current state from the enclave, and add it the DB"""
+    return fetch_object_and_add_to_db(con, 'concept_set_version_item', 'OmopConceptSetVersionItem', object_id)
+
+
+def concept_expression_enclave_to_db(con: Connection, object_id: str) -> Dict:
+    """Alias for: concept_set_version_item_enclave_to_db()"""
+    return concept_set_version_item_enclave_to_db(con, object_id)
+
+
+def concept_enclave_to_db(con: Connection, object_id: int) -> Dict:
+    """Given ID, get object's current state from the enclave, and add it the DB"""
+    return fetch_object_and_add_to_db(con, 'concept', 'OMOPConcept', object_id)
+
+
+# TODO: @Siggie: This function probably doesn't make sense there is no ConceptSetMembers Object API. Only a table.
+#  I think we need to udpate concept_set_members when we do a refresh, but I'm not fully understanding how to do it.
+def concept_set_members_enclave_to_db(con: Connection, codeset_id: int, concept_id: int, members_table_only=False):
```

Comment:
@Siggie: This function probably doesn't make sense there is no ConceptSetMembers Object API. Only a table. I think we need to update `concept_set_members` when we do a refresh, but I'm not fully understanding how to do it.